docker-compose.dev.yml

```
@@ -0,0 +1,20 @@
+    depends_on: [couch]
+  couch:
+    image: couchdb:3
+    environment:
+      - COUCHDB_USER=${COUCHDB_USER:-admin}
+      - COUCHDB_PASSWORD=${COUCHDB_PASSWORD:-adminpass}
+    ports: ["5984:5984"]
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:5984/"]
+      interval: 5s
+      timeout: 3s
+      retries: 20
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)